### DISCUSSION TITLE
How to make a language specific feed?

### DISCUSSION THREAD

**Discussion by daviga** [datenstrom/community#104] [Ask a question] [answered]
It would be good for multi language sites where languages live in their respective folders to have a feed/language (i.e. folder). In this way users won't receive feeds in languages they don't speak.

Just a suggestion.

**Comment by markseuffert** [comment, marked as answer]
Hi David,

I gave you the wrong answer per email, sorry. The multi language mode works exactly like you suggested and you don't need to change anything. If you enable it you get a feed per language (actually per folder) and not for the whole site. I am not using the multi language mode myself and forgot about that.

**Comment by daviga** [comment]
Well, that's good news. 

Thank you. I should have checked this my self.
